### PR TITLE
docs: examples with recorded output

### PR DESCRIPTION
**Pull request by guyellis**
Not sure if this is of value? I find examples like this useful to run from the command line to get an idea of how something functions.

### CODE REVIEW

File: example/counter.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+// Counter
+// Single Label
+// Multiple Values
+
+const register = require('..').register;
+
+const Counter = require('..').Counter;
```
Comment (SimenB) [inline example/counter.js:9]:
```suggestion
const { Counter, register } = require('..');
```

File: example/default-metrics.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+const register = require('..').register;
+
+require('..').collectDefaultMetrics({
```
Comment (SimenB) [inline example/default-metrics.js:5]:
```suggestion
const { collectDefaultMetrics, register } = require('..');

collectDefaultMetrics({
```

File: example/gauge.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// Gauge
+// Single Label
+// Multiple Values
+
+const register = require('..').register;
+
+const Gauge = require('../').Gauge;
```
Comment (SimenB) [inline example/gauge.js:9]:
```suggestion
const { Gauge, register } = require('..');
```

Comment (SimenB) [reply example/gauge.js:9]: etc 😅

**Review by SimenB — APPROVED** [review]
I like it. @siimon @zbjornson any reason not to include these?

**Comment by guyellis** [comment]
@SimenB @siimon @zbjornson - I've rebased this PR and I think that I've fixed all those requires as per @SimenB comments.

File: README.md [hunk 4]
```diff
@@ -104,22 +104,6 @@ const collectDefaultMetrics = client.collectDefaultMetrics;
...
-opposed to HTTP request durations observed continuously in a histogram), you
-should provide a `collect()` function, which will be invoked when Prometheus
-scrapes your metrics endpoint. `collect()` can either be synchronous or return a
-promise. See **Gauge** below for an example. (Note that you should not update
-metric values in a `setInterval` callback; do so in this `collect` function
-instead.)
-
-See [**Labels**](#labels) for information on how to configure labels for all
-metric types.
-
```
Comment (zbjornson) [inline README.md:122]: Oops, this should stay here.

**Review by zbjornson — CHANGES_REQUESTED** [review]
> any reason not to include these?

Only reason might be that these could become stale when new metrics are added or changed, but otherwise good with me!

File: README.md (hunk 4 above)
Comment (guyellis) [reply README.md]: No idea why I did this - reverted! Good catch.

**Review by zbjornson — APPROVED** [review]
🍻 great, thank you!

**Pull request merged**